src/warc2zim/content_rewriting/css.py

```diff
@@ -1,20 +1,22 @@
+# pyright: reportGeneralTypeIssues=false, reportAttributeAccessIssue=false
```

Comment:
Why using per-file ignores here while we use inline everywhere else?

Reply:
Because:
- `process` function was full of type checking errors
- I was lazy to read tinycss2 source code and understand how to fix them

This is now fixed.